src/Symfony/Component/Messenger/Transport/Semaphore/Util/PlatformUtil.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Transport\Semaphore\Util;
+
+/**
+ * @author Cedrick Oka Baidai <okacedrick@gmail.com>
```

Comment:
Should probably be marked `@internal`

Reply:
Btw, isn't `\DIRECTORY_SEPARATOR === '\\'` enough? I'm not sure we need a method for this.